projects/swimlane/ngx-ui/src/lib/components/toolbar/toolbar.component.ts

```diff
@@ -83,7 +48,8 @@ export class ToolbarComponent {
   }
   constructor(private renderer: Renderer2, private elRef: ElementRef) {}
 
-  onMenuClicked(item, $event) {
+  onMenuClicked(item: ToolbarMenuItem, $event: any) {
```

Comment:
can we change this event and its emitter to be type `Event`?

Reply:
typed it up